packages/run-protocol/README.md

```diff
@@ -25,3 +25,99 @@ Each interest charging period (say daily) the actual debts in all vaults are aff
 - a debtSnapshot on the vault by which one can calculate the actual debt
 
 To maintain that the keys of vaults to liquidate are stable requires that its keys are also time-independent so they're recorded as a "normalized collateralization ratio", with the actual collateral divided by the normalized debt.
+
+## Reading data off-chain
```

Comment:
These docs look _very_ handy. Thanks!

Reply:
Thanks for https://gist.github.com/dckc/73205178bd44cf3f4bbcf7db0fc280a8#file-borrow-from-vault-js it's cribbed from :)